### ISSUE TITLE
update openshmem readme?

### ISSUE THREAD

**Issue by hppritcha** [openshmem-org/openshmem#16]
Looks like the [README](https://github.com/openshmem-org/openshmem/blob/master/READMEl) might need updating with respect to contact info.

**Comment by tonycurtis** [comment]
> On Feb 26, 2016, at 11:20 AM, Howard Pritchard <notifications@github.com> wrote:
> 
> Looks like the README <https://github.com/openshmem-org/openshmem/blob/master/READMEl> might need updating with respect to contact info.
> 

Yes indeed (although, it’s still technically true; the best kind of true).  We’re in the process of updating contact info throughout at this very moment.

tony
